manifest.json

```diff
@@ -13,12 +13,11 @@
     }
   },
 
-  "permissions": ["management"],
+  "permissions": [],
```

Comment:
https://github.com/mozilla-ion/ion-basic-study/blob/main/ion.js#L58 probably still needs the `management` permission. That said, it would be nice if ion studies didn't need that permission.

Reply:
This PR grew bigger, so I created #14 to deal with this. Yes, we'll get rid of the "management" permission.